enderio-base/src/main/java/com/enderio/base/EnderIO.java

```diff
@@ -37,6 +47,8 @@ public EnderIO() {
         // Run datagen after registrate is finished.
         modEventBus.addListener(EventPriority.LOWEST, this::gatherData);
         modEventBus.addListener(this::ModelLoaders);
+        modEventBus.addListener(this::registerBERS);
+        modEventBus.addListener(EIOCapabilityManager::register);
```

Comment:
Why are we manually registering each handler instead of using @Mod.EventBusSubscriber and @SubscribeEvent? Just a design query